boards/xtensa/intel_s1000_crb/Kconfig.defconfig

```diff
@@ -167,6 +167,12 @@ config GPIO_DW_0
 
 endif
 
+if USB_DFU_CLASS
```

Comment:
who really needs SPI? USB_DFU or Flash? This should depend on Flash rathe than USB I guess

Reply:
ok. I will change it